VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/InAppRecruitmentScreen/InAppRecruitmentScreen.tsx

```diff
@@ -0,0 +1,66 @@
+import { StackScreenProps } from '@react-navigation/stack/lib/typescript/src/types'
+import { useTranslation } from 'react-i18next'
+import React, { FC } from 'react'
+
+import { Box, ButtonTypesConstants, ClickForActionLink, LargePanel, LinkTypeOptionsConstants, TextView, VABulletList, VAButton } from 'components'
+import { HomeStackParamList } from 'screens/HomeScreen/HomeStackScreens'
+import { NAMESPACE } from 'constants/namespaces'
+import { a11yLabelVA } from 'utils/a11yLabel'
+import { useTheme } from 'utils/hooks'
+import getEnv from 'utils/env'
+
+type InAppRecruitmentScreenProps = StackScreenProps<HomeStackParamList, 'InAppRecruitment'>
+
+const { LINK_URL_IN_APP_RECRUITMENT, LINK_URL_VETERAN_USABILITY_PROJECT } = getEnv()
+
+const InAppRecruitmentScreen: FC<InAppRecruitmentScreenProps> = ({ navigation }) => {
+  const { t } = useTranslation(NAMESPACE.COMMON)
+  const theme = useTheme()
+
+  const onPress = () => {
+    navigation.navigate('Webview', { url: LINK_URL_IN_APP_RECRUITMENT, displayTitle: t('webview.vagov'), loadingMessage: t('inAppRecruitment.goToQuestionnaire.loading') })
+  }
+
+  return (
+    <LargePanel title={t('inAppRecruitment.giveFeedback')} rightButtonText={t('close')}>
+      <Box mt={theme.dimensions.contentMarginTop} mb={theme.dimensions.contentMarginBottom} mx={theme.dimensions.gutter}>
+        <TextView variant="MobileBodyBold" accessibilityRole="header">
+          {t('inAppRecruitment.makeAppBetter.header')}
+        </TextView>
+        <TextView variant="MobileBody" mt={theme.dimensions.standardMarginBetween} paragraphSpacing={true}>
```

Comment:
```suggestion
        <TextView variant="MobileBody" my={theme.dimensions.standardMarginBetween}>
```

This makes the margin above the bullet point list match the Figma mock

<img width="551" alt="image" src="https://github.com/department-of-veterans-affairs/va-mobile-app/assets/9042882/0f36b358-571d-4d44-a578-def89616b350">

Since Dylan is out, I'll let others confirm before I commit this

Reply:
committed, yeah since bullet points are afterwards it doesn't need the paragraph spacing and figma is correct.